### PR TITLE
Rekor cli extension

### PR DESCRIPTION
**Pull request by kdacosta0**

### CODE REVIEW

File: test/cosign/cosign_sign_verify_test.go [hunk 1]
```diff
@@ -77,14 +118,82 @@ var _ = Describe("Cosign test", Ordered, func() {
...
+		It("should verify the signature and extract logIndex", func() {
+			output, err := cosign.CommandOutput(testsupport.TestContext, "verify", "--certificate-identity-regexp", ".*@redhat", "--certificate-oidc-issuer-regexp", ".*keycloak.*", targetImageName)
+			Expect(err).ToNot(HaveOccurred())
+
+			logrus.Info(string(output))
+
+			startIndex := strings.Index(string(output), "[")
+			if startIndex == -1 {
+				// Handle error: JSON start not found
+				return
```
Comment (bouskaJ) [inline test/cosign/cosign_sign_verify_test.go:135]: Better to fail the test in this case.

File: test/cosign/cosign_sign_verify_test.go [hunk 2]
```diff
@@ -77,14 +118,82 @@ var _ = Describe("Cosign test", Ordered, func() {
...
+
+			output, err := rekorCli.CommandOutput(testsupport.TestContext, "get", "--rekor_server", rekorServerURL, "--log-index", logIndexStr)
+			Expect(err).ToNot(HaveOccurred())
+
+			logrus.Info(string(output))
+
+			startIndex := strings.Index(string(output), "{")
+			if startIndex == -1 {
+				// Handle error: JSON start not found
+				return
```
Comment (bouskaJ) [inline test/cosign/cosign_sign_verify_test.go:160]: Better to fail the test in this case.

**Review by bouskaJ — COMMENTED** [review]
@kdacosta0 great work!

File: test/cosign/cosign_sign_verify_test.go [hunk 3]
```diff
@@ -19,12 +24,45 @@ import (
 
 const testImage string = "alpine:latest"
 
+var logIndex int
+var hashValue string
+var tempDir string
+var publicKeyPath string
+var signaturePath string
+
+type CosignVerificationOutput []struct {
```
Comment (osmman) [inline test/cosign/cosign_sign_verify_test.go:33]: I think that these `*Output` structure should be moved to some support package and be reusable by other tests/

Comment (kdacosta0) [reply test/cosign/cosign_sign_verify_test.go]: moved them to testsupport package

File: test/cosign/cosign_sign_verify_test.go (hunk 1 above)
Comment (kdacosta0) [reply test/cosign/cosign_sign_verify_test.go]: done

**Review by osmman — APPROVED** [review]

**Pull request merged**